src/main/java/io/candydoc/ddd/extract_ddd_concepts/NoBoundedContextNorSharedKernelFound.java

```diff
@@ -0,0 +1,13 @@
+package io.candydoc.ddd.extract_ddd_concepts;
+
+import io.candydoc.ddd.model.ExtractionException;
+import java.util.List;
+
+public class NoBoundedContextNorSharedKernelFound extends ExtractionException {
+  public NoBoundedContextNorSharedKernelFound(List<String> packageToScans) {
+    super(
+        "No bounded context or shared kernel has been found in this packages : '"
```

Comment:
Use String.format to variabilize the string and pass packageToScans as a parameter:

```
String.format("No bounded context or shared kernel has been found in this packages : '%s'., packageToScans)
```